assets/tests/components/detours/diversionPage.delete.test.tsx

```diff
@@ -0,0 +1,133 @@
+import { beforeEach, describe, expect, jest, test } from "@jest/globals"
+
+import React from "react"
+
+import { screen } from "@testing-library/dom"
+import "@testing-library/jest-dom/jest-globals"
+import { render } from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+
+import { DetourListPage } from "../../../src/components/detourListPage"
+import {
+  deleteDetour,
+  fetchDetour,
+  fetchDetours,
+  putDetourUpdate,
+} from "../../../src/api"
+import { Ok } from "../../../src/util/result"
+import { neverPromise } from "../../testHelpers/mockHelpers"
+import getTestGroups from "../../../src/userTestGroups"
+import { detourListFactoryWithDraft } from "../../factories/detourListFactory"
+import { TestGroups } from "../../../src/userInTestGroup"
+import {
+  cancelButton,
+  confirmDeleteDetourButton,
+  deleteDetourButton,
+} from "../../testHelpers/selectors/components/detours/diversionPage"
+import { draftDetourFactory } from "../../factories/detourStateMachineFactory"
+
+jest
+  .useFakeTimers({ doNotFake: ["setTimeout"] })
+  .setSystemTime(new Date("2024-08-29T20:00:00"))
+
+jest.mock("../../../src/userTestGroups")
+
+jest.mock("../../../src/api")
+
+beforeEach(() => {
+  jest.mocked(fetchDetours).mockReturnValue(neverPromise())
+  jest.mocked(fetchDetour).mockReturnValue(neverPromise())
+  jest.mocked(putDetourUpdate).mockReturnValue(neverPromise())
+  jest.mocked(deleteDetour).mockReturnValue(neverPromise())
+  jest
+    .mocked(getTestGroups)
+    .mockReturnValue([
+      TestGroups.DetoursPilot,
+      TestGroups.DetoursList,
+      TestGroups.DeleteDraftDetours,
+    ])
+})
+
+describe("Detours Page: Open a Detour", () => {
+  test("calls API with correct detour ID", async () => {
+    jest
+      .mocked(fetchDetours)
+      .mockResolvedValue(Ok(detourListFactoryWithDraft.build()))
+
+    render(<DetourListPage />)
+
+    await userEvent.click(await screen.findByText("Draft Detour 123"))
+    expect(fetchDetour).toHaveBeenCalledWith(123)
+  })
```

Comment:
question: do you think we need this test if this specific case is covered by `detoursListPage.openDetour.test.tsx`?